final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +76,78 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = {
+  "name" => '',
+  "smell" => '',
+  "weight" => 400,
+  "cities_destroyed" => ['Paris', 'Milan', 'Texas'],
+  "lucky_numbers" => [1, 2, 3, 4],
+  "address" => {
+    "number" => 10,
+    "street" => 'King',
+    "state" => 'CO',
+    "zip" => 80301
+  }
+}
+
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
-
+#   - energy_level = 50
+class SuperHero
+  attr_accessor :name, :super_power, :age
+  attr_reader :arch_nemesis, :power_level, :energy_level
+
+  def initialize(name, super_power, age, power_level, energy_level)
+    @name = name
+    @super_power = super_power
+    @age = age
+    @arch_nemesis = arch_nemesis
```

Comment:
For lines 111-113, check the directions written up on lines 99 about these needing to be static values and adjust your solution. Let me know if you need help or have questions about that.